vhdl/quartus/res/components.ipx

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Just looking forward to the full autogen design review here: 
could we put the ipx file in a common folder that's shared between all Quartus projects, whether they were autogened or not? E.g.
```
de10_nano_projects
|
|---common
     |--- components.ipx
|--- project1
|--- project2
...
```